spec/core_functions/meta/accepts_content.hrx

```diff
@@ -0,0 +1,77 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#2807
+
+<===>
+================================================================================
+<===> accepts/direct-child/input.scss
+@use "sass:meta";
+
+@mixin foo() {
+    @content;
```

Comment:
[Use two spaces for indentation](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-use-two-spaces-for-indentation)

Reply:
Resolved. It would be nice if `lint-spec` enforced this or if `lint-spec --fix` fixed this -- it was somewhat tedious to grep for `    ` to find all places with the wrong indentation.